servers/services/src/main/java/org/projectnessie/services/impl/ConfigApiImpl.java

```diff
@@ -33,9 +34,18 @@ public ConfigApiImpl(ServerConfig config, VersionStore store) {
 
   @Override
   public NessieConfiguration getConfig() {
```

Comment:
Is `RepositoryInformation` supposed to change throughout the lifecycle of a Nessie instance? It seems that it is not, except maybe for the method `getAdditionalProperties()`.

If this information doesn't change, then `NessieConfiguration` could be cached in application scope. I would therefore like to suggest something: `NessieConfiguration` could be exposed as a singleton CDI bean, then injected here – rather than having one instance created here, per request.

The reason why I'm suggesting this is that this bean is likely to be useful in the events workflow as well.

Reply:
`RepositoryInformation` is a function of the repo ID, which is currently fixed, but may not always be fixed (cf. #5342).